WordPress/src/main/java/org/wordpress/android/ui/prefs/language/LocalePickerBottomSheet.kt

```diff
@@ -0,0 +1,181 @@
+package org.wordpress.android.ui.prefs.language
+
+import android.os.Bundle
+import android.view.KeyEvent
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.view.inputmethod.EditorInfo
+import android.widget.FrameLayout
+import androidx.core.widget.doOnTextChanged
+import androidx.lifecycle.ViewModelProvider
+import androidx.recyclerview.widget.DividerItemDecoration
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.google.android.material.bottomsheet.BottomSheetBehavior
+import com.google.android.material.bottomsheet.BottomSheetDialog
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import org.wordpress.android.WordPress
+import org.wordpress.android.databinding.LocalePickerBottomSheetBinding
+import org.wordpress.android.util.ActivityUtils
+import javax.inject.Inject
+
+class LocalePickerBottomSheet : BottomSheetDialogFragment() {
+    @Inject lateinit var viewModelFactory: ViewModelProvider.Factory
+    private lateinit var viewModel: LocalePickerViewModel
+
+    private val localeAdapter = LocalePickerAdapter()
+
+    private var binding: LocalePickerBottomSheetBinding? = null
+
+    private var bottomSheet: FrameLayout? = null
+
+    private var callback: LocalePickerCallback? = null
+
+    fun setLocalePickerCallback(callback: LocalePickerCallback?) {
+        this.callback = callback
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        (requireActivity().applicationContext as WordPress).component().inject(this)
+        viewModel = ViewModelProvider(requireActivity(), viewModelFactory).get(LocalePickerViewModel::class.java)
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        binding = LocalePickerBottomSheetBinding.inflate(inflater, container, false)
+        return binding?.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        binding?.apply {
+            setupContentViews()
+            setupObservers()
+        }
+    }
+
+    private fun LocalePickerBottomSheetBinding.setupContentViews() {
+        val layoutManager = LinearLayoutManager(context)
```

Comment:
LayoutManager is also set in layout xml, hence these two lines are not necessary here

Reply:
Ah, good catch 👍